File: indicator/src/main/java/com/zhpan/indicator/base/IIndicator.java
package com.zhpan.indicator.base;

import androidx.viewpager.widget.ViewPager;

import com.zhpan.indicator.model.IndicatorOptions;


/**
 * <pre>
 *   Created by zhangpan on 2019-09-02.
 *   Description:
 * </pre>
 */
public interface IIndicator extends ViewPager.OnPageChangeListener{

    void setPageSize(int pageSize);

    void setIndicatorOptions(IndicatorOptions options);

}

File: indicator/src/main/java/com/zhpan/indicator/base/IIndicator.kt
package com.zhpan.indicator.base

import androidx.viewpager.widget.ViewPager

import com.zhpan.indicator.option.IndicatorOptions


/**
 * <pre>
 * Created by zhangpan on 2019-09-02.
 * Description:
</pre> *
 */
interface IIndicator : ViewPager.OnPageChangeListener {

    fun notifyDataChanged()

    fun setIndicatorOptions(options: IndicatorOptions)

}
